server/routerlicious/packages/services-shared/src/test/restLess.spec.ts

```diff
@@ -36,17 +35,24 @@ describe("RestLess", () => {
                     .catch(next);
             };
         };
-        app.use(restLessMiddleware());
         // set up rudimentary authentication
         const authMiddleware: () => express.RequestHandler = () => (req, res, next) => {
             if (req.get("Authorization") !== `Bearer ${authToken}`) {
                 return res.sendStatus(403);
             }
             next();
         };
-        app.use(authMiddleware());
+        if (restLessBeforeBodyParser) {
```

Comment:
Educate me: are we introducing these 2 modes of tests depending on the value of `restLessBeforeBodyParser` just to improve our test coverage, or is this also something related to `formidable`?

Reply:
This is to improve test coverage. Only 1 path of RestLess server was being tested (the `formidable` path). So I added this alternate variation to test the bodyparser path. All of our services use the formidable path, which is good, because you can see here I had to make a change to urlencoded to allow it to recognize a restless form body.